interface/src/app/analyze/analysis/analysis.js

```diff
@@ -57,6 +57,10 @@ AnnotationType, SampleBin) {
     });
   };
 
+  $scope.showVolcanoPlot = function() {
+    $state.go('volcano');
```

Comment:
Ok, as I said this is good getting filed under an issue for now. I also saw your TODO comments in volcano-plot/view.js. Ok, cool!